CMakeLists.txt

```diff
@@ -257,126 +257,95 @@ else()
     if(NOT UMF_LINK_HWLOC_STATICALLY)
         pkg_check_modules(LIBHWLOC hwloc>=2.3.0)
         if(NOT LIBHWLOC_FOUND)
-            find_package(LIBHWLOC 2.3.0 REQUIRED hwloc)
+            find_package(LIBHWLOC 2.3.0 COMPONENTS hwloc)
+            if(LIBHWLOC_LIBRARIES)
+                set(LIBHWLOC_AVAILABLE TRUE)
```

Comment:
nit: do we need this extra `LIBHWLOC_AVAILABLE` var? you could use in the `if` below simply `LIBHWLOC_LIBRARIES`

Reply:
I've added this variable to increase readability of this logic. This could be replaced in the future with `LIBHWLOC_FOUND` variable set in hwloc Find module: #1192 .